docs/local-with-internet-identity.md

```diff
@@ -0,0 +1,39 @@
+# Local Development with internet-identity
+
+Follow these steps to run cancan locally along with locally running [dfinity/internet-identity](https://github.com/dfinity/internet-identity) canisters on the same replica.
+
+## Overview
+
+The internet-identity canisters must be running on the same IC replica as the CanCan canisters.
+
+Because CanCan's dfx.json configures its replica to run on port 9000, we need to be sure to install the local internet-identity to the replica running on that port.
+
+CanCan must then be accessed in such a way that it uses the local internet-identity canisters to log in instead of the product internet-identity canisters.
+
+## Steps
+
+1. Follow the CanCan [README](../README.md) to start a local replica, deploy CanCan, and opn the app to the sign in screen.
+2. Clone the git repo for https://github.com/dfinity/internet-identity to another directory
+3. Ensure you have the correct dependencies to build internet-identity (e.g. `rustc@1.51.0` at time of this writing)
+4. Add a network called `local-cancan` to the internet-identity `dfx.json` that corresponds to the same network that cancan runs on (the `local` network in cancan dfx.json).
```

Comment:
You could just change the port in `cancan`’s `dfx.json` to the `8000` that is the default for dfx projects, then this step wouldn’t be necessary and both could use `local`

Reply:
We're going to do that next. But just want these docs to be accurate for how it works so far. I will update them with work on https://dfinity.atlassian.net/browse/CAN-127